vqpy/operator/tracker/byte_tracker.py

```diff
@@ -23,14 +23,14 @@ def __init__(self, data: Dict):
             self.track_id = self.next_id()
             # TODO: remove unnecessary track_id assignments
             self.data = data
-            self._tlbr = np.asarray(data["tlbr"], dtype=np.float)
+            self._tlbr = np.asarray(data["tlbr"], dtype=float)
```

Comment:
Why do we change the type?

Reply:
numpy v1.20.0 deprecated alias for builtin `float`: `float` should be used instead of the alias `np.float`.